apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -278,6 +279,8 @@ const EventTypePage = (props: EventTypeSetupProps) => {
               .filter((slug) => slug !== eventType.slug) ?? [],
         },
       })),
+      multipleDurationEnabled: eventType.metadata?.multipleDuration || false,
+      seatsPerTimeSlotEnabled: eventType.seatsPerTimeSlot,
```

Comment:
Enable this default value as you had to navigate to the "Advanced" tab for this value to appear